<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainUI</class>
 <widget class="QWidget" name="TerrainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Court Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel_2">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rankingLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="typeLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="courtsTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addCourtButton">
       <property name="text">
        <string>Add Court</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCourtButton">
       <property name="text">
        <string>Delete Court</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateCourtButton">
       <property name="text">
        <string>Update Court</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>